buddyboss/buddyboss-platform
# #1227 - Fix: Inconsistent video preview for forum activity feed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1227  .

### How to test the changes in this Pull Request:

1. Make sure that link previews are enabled on forums and activity feed.
2. Compose your new Discussion with a YouTube link.
3. Now, go to any discussion and create a new reply with YouTube URL for more check video.
4. Go to the activity feed and can see the video preview the same as discussion/reply.


### Proof Screenshots or Video
https://vimeo.com/619810050/9bcb18337b


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
